src/main/java/org/eclipse/golo/runtime/AugmentationMethodFinder.java

```diff
@@ -129,7 +132,7 @@ public MethodHandle find(Class<?> definingModule, Class<?> augmentedClass) {
     protected String augmentationClassName(Class<?> definingModule, String augmentationName) {
       int idx = augmentationName.lastIndexOf(".");
       if (idx == -1) { return augmentationName; }
-      return (new StringBuilder(augmentationName)).replace(idx, idx+1, "$").toString();
+      return (new StringBuilder(augmentationName)).replace(idx, idx + 1, "$").toString();
```

Comment:
I wonder if it is a good idea to use `StringBuilder` instead of `String`. Not a big deal, though.